docs/development/architecture.rst

```diff
@@ -56,7 +56,7 @@ For the simplicity the other execution systems - Dask and MPI are omitted and on
   on a selected storage format and mapping or compiling the Dataframe Algebra DAG to and actual
```

Comment:
The line `.. _Arrow tables: https://arrow.apache.org/docs/python/generated/pyarrow.Table.html` should be remove in this file.

Reply:
removed